scan/libmailgoose/scan.py

```diff
@@ -567,7 +567,16 @@ def scan_dkim(
 
         selector_full_name = signature_tags[b"s"] + b"._domainkey." + signature_tags[b"d"] + b"."
         if dkim_record_raw := dkim.dnsplug.get_txt(selector_full_name):
-            dkim_record_tags = dkim.util.parse_tag_value(dkim_record_raw)
+            try:
+                dkim_record_tags = dkim.util.parse_tag_value(dkim_record_raw)
+            except dkim.util.InvalidTagSpec as e:
+                raise dkim.DKIMException(
+                    "The DKIM DNS record contains an invalid tag: " + e.args[0].decode("ascii", errors="ignore")
```

Comment:
I believe that `.decode("ascii", errors="replace")` would be better. It will shpw the user that there was something there:
```
>>> "ązzć".encode().decode("ascii", errors="ignore")
'zz'
>>> "ązzć".encode().decode("ascii", errors="replace")
'��zz��'
>>> "ćóś".encode().decode("ascii", errors="ignore")
''
>>> "ćóś".encode().decode("ascii", errors="replace")
'������'
```